webpack.dev.ts

```diff
@@ -23,7 +23,7 @@ module.exports = mergeDev(commonDev, {
     poll: 1000,
   },
   devServer: {
-    hot: true,
+    hot: false,
```

Comment:
😱 ??

Reply:
Before I say anything, this isn't necessarily going to stay. I was just experimenting with settings.

Anyway, .I was reading the docs, and this bit from the document on [Hot Module Replacement](https://webpack.js.org/concepts/hot-module-replacement/) caught my eye (emphasis mine):

>Hot Module Replacement (HMR) exchanges, adds, or removes [modules](https://webpack.js.org/concepts/modules/) while an application is running, **without a full reload.** This can significantly speed up development in a few ways...

As long as I've been developing our application, it's done full reloads, not module replacement. This got me thinking - what if our dev config is misconfigured or not working as intended? As far as I can tell, changing `hot` to `false` does not change the behavior of the application in development mode when it comes to changing a piece of code.